src/screens/ProfileScreen/elements/LinkAndSocialMedia.tsx

```diff
@@ -40,153 +28,83 @@ const Button: React.FC<ButtonProps> = ({onPress, style, children}) => {
-      social: Share.Social.TWITTER,
-      failOnCancel: true
-    };
-
-    try {
-      await Share.shareSingle(shareOptions);
-    } catch (error) {
-      console.log('Error =>', error);
-    }
-  };
-
-  return (
-    <View style={styles.linkAndSocialMediaContainer}>
-      <Text style={styles.linkAndSocialMediaTitle}>Receive anonymous messages anywhere:</Text>
-      <View style={styles.shareStepContainer}>
-        <Text style={styles.shareStepLabel}>Step 1: Copy your link</Text>
+    <View
+      style={{
+        borderWidth: 1,
+        borderColor: COLORS.gray,
+        borderRadius: 10,
+        marginTop: 19,
+        padding: 12
+      }}>
+      <Text style={styles.linkAndSocialMediaTitle}>Receive anonymous messages anywhere!</Text>
+      <Text
+        style={{
+          color: COLORS.gray500,
+          fontWeight: '400',
+          fontSize: 12,
+          paddingVertical: 8,
+          lineHeight: 18
+        }}>
+        Share your link on Insta, TikTok or elsewhere to receive compliments, questions or feedback
+        from friends.
+      </Text>
+      <View
+        style={{
+          display: 'flex',
+          flexDirection: 'row',
+          justifyContent: 'space-between'
+        }}>
         <CopyLink username={username} />
-      </View>
-
-      <View style={styles.shareStepContainer}>
-        <Text style={styles.shareStepLabel}>Step 2: Share your link</Text>
-
-        <View style={{flexDirection: 'row'}}>
-          <Button onPress={shareTwitter} style={styles.tweetButton}>
-            <Text style={styles.buttonSocialMediaLabel}>Tweet</Text>
-            <TwitterIcon height={16} width={20} />
-          </Button>
-
-          {/* <InstagramButton socialMessage={socialMessage} /> */}
-          <View style={{width: 10}} />
-
-          <Button
-            onPress={() => {
-              ShareUtils.shareUserLink(username);
-            }}
-            style={styles.elseWhereButton}>
-            <Text ellipsizeMode="tail" numberOfLines={1} style={styles.elseWhereLabel}>
-              Elsewhere
+        <TouchableOpacity
+          onPress={() => {
+            ShareUtils.shareUserLink(username);
+          }}
+          style={{
+            backgroundColor: COLORS.signed_primary,
+            height: 34,
+            borderRadius: 8,
+            width: '30%'
+          }}>
+          <View
+            style={{
+              display: 'flex',
+              flexDirection: 'row',
+              justifyContent: 'center',
+              alignItems: 'center',
+              marginVertical: 8,
+              marginHorizontal: 10
+            }}>
+            <ShareIcon />
+            <Text
+              style={{
+                fontSize: 12,
+                color: COLORS.white,
+                marginLeft: 8
+              }}>
+              Share
             </Text>
-          </Button>
-        </View>
+          </View>
+        </TouchableOpacity>
       </View>
     </View>
   );
 };
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The new implementation of `LinkAndSocialMedia` has removed the Instagram and Twitter sharing buttons. If this was intentional and these features are no longer needed, then the changes look good. However, if this was an oversight, you may want to reintroduce these features.



<!-- This is an auto-generated comment by OSS CodeRabbit -->